<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchemaEditorForm</class>
 <widget class="QWidget" name="SchemaEditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Schema Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../libs/libgui/res/resources.qrc">
    <normaloff>:/icons/icons/pgsqlModeler48x48.png</normaloff>:/icons/icons/pgsqlModeler48x48.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="horizontalSpacing">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Editor</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QToolButton" name="new_sch_tb">
         <property name="text">
          <string>New</string>
         </property>
         <property name="icon">
          <iconset resource="../../../libs/libgui/res/resources.qrc">
           <normaloff>:/icons/icons/new.png</normaloff>:/icons/icons/new.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="save_sch_tb">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
         <property name="icon">
          <iconset resource="../../../libs/libgui/res/resources.qrc">
           <normaloff>:/icons/icons/save.png</normaloff>:/icons/icons/save.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="5">
        <widget class="QTabWidget" name="editors_tbw">
         <property name="tabPosition">
          <enum>QTabWidget::South</enum>
         </property>
         <property name="tabsClosable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="save_as_sch_tb">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Save as</string>
         </property>
         <property name="icon">
          <iconset resource="../../../libs/libgui/res/resources.qrc">
           <normaloff>:/icons/icons/saveas.png</normaloff>:/icons/icons/saveas.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>322</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="load_sch_tb">
         <property name="text">
          <string>Load</string>
         </property>
         <property name="icon">
          <iconset resource="../../../libs/libgui/res/resources.qrc">
           <normaloff>:/icons/icons/open.png</normaloff>:/icons/icons/open.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Syntax</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <property name="spacing">
        <number>10</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="syntax_conf_wgt" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="save_conf_tb">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Save the changes in syntax highlighting settings to the conf file.</string>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
         <property name="icon">
          <iconset resource="../../../libs/libgui/res/resources.qrc">
           <normaloff>:/icons/icons/save.png</normaloff>:/icons/icons/save.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="apply_conf_tb">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Apply the loaded settings to the open editors.</string>
         </property>
         <property name="text">
          <string>Apply</string>
         </property>
         <property name="icon">
          <iconset resource="../../../libs/libgui/res/resources.qrc">
           <normaloff>:/icons/icons/confirm.png</normaloff>:/icons/icons/confirm.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QWidget" name="syntax_wgt" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../libs/libgui/res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
